### PR TITLE
cleanup: remove cmd assert librarys

### PR DESCRIPTION
**Pull request by demoManito**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
remove assert libraries, avoid the use of assert libraries to help your tests, so I want to refactor the tests that are done using assertions (https://github.com/golang/go/wiki/TestComments#assert-libraries)

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/approve
for cmd/kubeadm

File: cmd/kubeadm/app/phases/certs/certs_test.go [hunk 1]
```diff
@@ -739,15 +739,25 @@ func TestNewCSR(t *testing.T) {
 		t.Errorf("invalid signature on CSR: %v", err)
 	}
 
-	assert.ElementsMatch(t, certConfig.Organization, csr.Subject.Organization, "organizations not equal")
+	sort.Strings(certConfig.Organization)
```
Comment (lavalamp) [inline cmd/kubeadm/app/phases/certs/certs_test.go:742]: This modifies the object under test, the old line did not. I don't know if this is important, but this is a common-ish way to get a bug in a test.

**Comment by lavalamp** [comment]
What is the benefit of doing this? I don't think we should make changes like this unless there's an agreed-upon project-wide ban of these libraries; it's more effort to be correct than it is worth to change.

I skimmed only briefly and found at least one place where the new code did something different than the old code.

**Comment by demoManito** [comment]
> What is the benefit of doing this? I don't think we should make changes like this unless there's an agreed-upon project-wide ban of these libraries; it's more effort to be correct than it is worth to change.
> 
> I skimmed only briefly and found at least one place where the new code did something different than the old code.

hello @lavalamp
1. golang does not recommend using the assert library in testing. 
2. In the cmd package, there are relatively few places to use the assertion library. Most of them use the default method (`if err!=nil {t.Error (err)}`). I want to unify the test code style under the cmd package.

**Comment by endocrimes** [comment]
/remove-sig node

This doesn't touch the kubelet, but: I'd air on the side of rejecting this PR, as much as it would be nice to unify styles over time, change for the sake of change doesn't help us make progress, but it does take a lot of review cycles. It would also be a much broader discussion to settle on a default path vs when exceptions make sense.

**Comment by lavalamp** [comment]
I think things like this should be accomplishing some goal set by sig arch code organization subproject. So, the way forward is to get them to agree that the benefit is worth the cost; I think it is probably not. But please feel free to reopen this if you get them to agree.

**Pull request closed without merging**